### ISSUE TITLE
Download files in a row

### ISSUE THREAD

**Issue by PRosenb** [aria2/aria2#1019]
Hi all,
Is it possible to configure area2 in a way that it downloads all files from the same server in a row? This means it downloads one file until it is finished and then starts the next one.
Thanks guys.

**Comment by tatsuhiro-t** [comment]
Yes, if you know all URIs you want to download.  Write them down in a file, and feed it to aria2c.  Use -j1 to sequentially download a file one by one.

**Comment by PRosenb** [comment]
Many thanks @tatsuhiro-t for your reply, that's a good idea.
I'm using Aria2 WebUI and this can be done there too with adding all files at once. It is not possible though to add more files after download has started.
Is there also a global option to achieve this so that I could add files anytime?
